src/transformers/models/albert/modeling_tf_albert.py

```diff
@@ -146,7 +146,7 @@ def __init__(self, config: AlbertConfig, **kwargs):
         self.LayerNorm = tf.keras.layers.LayerNormalization(epsilon=config.layer_norm_eps, name="LayerNorm")
         self.dropout = tf.keras.layers.Dropout(rate=config.hidden_dropout_prob)
 
-    def build(self, input_shape: tf.TensorShape):
+    def build(self, input_shape=None):
```

Comment:
Nit - shouldn't this become an optional? 

```suggestion
    def build(self, input_shape: Optional[tf.TensorShape] = None):
```